code/modules/spells/spell_types/conjure.dm

```diff
@@ -56,6 +56,18 @@
 	range = 3
 	newVars = list("emagged" = 2, "remote_disabled" = 1,"shoot_sound" = 'sound/weapons/laser.ogg',"projectile" = /obj/item/projectile/beam/laser, "declare_arrests" = 0,"name" = "Wizard's Justicebot")
 
+/obj/effect/proc_holder/spell/aoe_turf/conjure/linkWorlds
+	name = "Link Worlds"
+	desc = "A whole new dimension for you to play with! They won't be happy about it, though."
+	invocation = "WTF"

```

Comment:
Please try harder

Reply:
whatever, it's an admin spell that's completely unbalanced.

Reply:
i mean look above that spawns like 6 securitrons with magic missiles if i remember correctly